test/e2e/knit_irqaff_test.go

```diff
@@ -12,14 +12,14 @@ import (
 
 var _ = g.Describe("knit IRQ affinity tests", func() {
 
-	var fixtureName = "xeon-multinuma-00"
+	var fixtureName = "cluster_bot_411"
 
 	var (
 		dataDir      string
 		snapshotRoot string
 	)
 
-	g.Context("Without isolated CPUs", func() {
+	g.Context("With isolated, reserved and offlined CPUs", func() {
```

Comment:
not a fan of this. It's fine to have more data with different config, but then better to have more tests rather than retargeting the existing one

Reply:
I've change it because is not possible anymore to add a test without isolated CPU's then I had to add a realistic one